src/lib/utils/utils.ts

```diff
@@ -3,6 +3,18 @@
 import Prando from 'prando';
 
 const utils = {
+  convertCelsiusToFahrenheit: function (celsius: number): number {
+    // semamamama please fill this in
+    (celsius * 9) / 5 + 32;
```

Comment:
Your tests are failing, youre doing the calculation correct however youre not returning the value properly

Fix your return value for both functions, and run yarn test on your terminal to ensure tests passes